src/GafferScene/StandardAttributes.cpp

```diff
@@ -69,6 +69,10 @@ StandardAttributes::StandardAttributes( const std::string &name )
 
 	attributes->addChild( new Gaffer::NameValuePlug( "filteredLights", new IECore::StringData( "" ), false, "filteredLights" ) );
 
+	// visualisers
```

Comment:
I think this belongs on OpenGLAttributes rather than StandardAttributes - that's where everything else specific to the viewer is. The general idea with StandardAttributes is that it contains attributes that could be applicable to _any_ renderer.

Reply:
Addressed in 20367a5. I've also renamed the section (now "Light Visualisers") and pefixed the attribute name with "gl:". Additionally, the default value is 512 now as per what the `GoboVisualiser` used.